gci/management/commands/gci_no_issue_tasks.py

```diff
@@ -0,0 +1,31 @@
+import os.path
+
+from ruamel.yaml import YAML
+from gci.gitorg import get_issue
+
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = 'Filter GCI tasks without issues'
```

Comment:
Can be more clear. e.g. `Extract non-issue tasks from GCI tasks.yaml and dump into non_issue_tasks.yaml`

Reply:
Maybe the file name is not worth mentioning here, but at least elaborate a bit more.